### ISSUE TITLE
viewing full text of log actions behind the ellipses is very difficult for screen reader users

### ISSUE THREAD

**Issue by Grahamwp** [UTRS/utrs#135]
I just went into UTRS for the first time today to check out a block relating to one of my actions, #18351. I found it impossible to view the text beyond the ellipses with my screen reader. I informed the reviewing admin, who told me that the full text is displayed in a new window that scrolls when clicked in. This is unhelpful for users like me because screen readers have no concept of scrolling in to anything ... many of them show a virtual representation of each webpage which can be somewhat different from how it looks to a sighted user. I tried it with all the combinations of screen readers/browsers I have (JAWS/NVDA and FF/IE) in Windows and I couldn't get it to work reliably. The only way I could view the full text of the log actions was to view the HTML source. Even something like an expand/expand all link near the log actions would make things much easier for me.

**Comment by NicosArv** [comment]
I don’t have a screen reader in use and cannot view the full text in the user action comments box. they end with “…” and no amount of clicking helps. This is in safari. A view more or some other button that expands the text is requested or otherwise clicking the text or box results in full text on all browsers. I am using Safari

**Comment by TParis** [comment]
Hi, did you click the link that says "logs for this request"?

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by NicosArv** [comment]
Yes @TParis and thank you. I did figure it out after pressing almost everything in the interface. Not immediately user friendly and the lack of a help section didn’t help.

**Comment by TParis** [comment]
Roger on the accessibility issues.  We are in the middle of working with a San Francisco non-profit to redesign UTRS and I’ll ask DeltaQuad to make sure she includes consideration for screen readers in the requirements.

v/r,
TParis

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________
